py-polars/tests/unit/io/test_excel.py

```diff
@@ -389,3 +391,12 @@ def test_excel_empty_sheet(empty_excel_file_path: Path) -> None:
 
     df = pl.read_excel(empty_excel_file_path, raise_if_empty=False)
     assert_frame_equal(df, pl.DataFrame())
+
+
+@pytest.mark.parametrize("hidden_columns", [["a"], ["a", "b"], cs.numeric(), cs.last()])
+def test_excel_hidden_columns(hidden_columns: list[str] | SelectorType) -> None:
```

Comment:
I didn't find method to read_excel but ignore the hidden columns, so this test may not be very good.

Reply:
Tthis is fine for the purposes of the test; I think it _may_ be possible to determine which columns are hidden using the new `openpyxl` engine, but it's not an option we currently expose (I might take a look at that later).